src/zope/testbrowser/browser.py

```diff
@@ -27,6 +27,7 @@
 from zope.cachedescriptors.property import Lazy
 from wsgiproxy.proxies import TransparentProxy
 from bs4 import BeautifulSoup
+from soupsieve import escape as css_escape
```

Comment:
NB: this needs soupsieve >= 1.9.0.

https://facelessuser.github.io/soupsieve/api/#soupsieveescape

(I should probably explicitly add soupsieve to setup.py's install_requires.)